tests/core/test_taskview.py

```diff
@@ -41,3 +44,15 @@ def test_no_detect_tags(self):
         matches = re.findall(TAG_REGEX, content)
 
         self.assertEqual([], matches)
+
+    def test_get_title(self):
+        task_title = 'Very important task'
+        task = Task(id = uuid4(), title=task_title)
+        view = TaskView(Datastore(), task, None, False)
```

Comment:
These 3 lines can probably be encapsulated into a `create_task_view_from_task()` test helper, but I can do that when I add another test.